package.json

```diff
@@ -14,7 +14,7 @@
     "dev": "tsc -w & nodemon ./dist/index.js",
     "lint": "echo Nothing to do for target 'lint'",
     "test": "jest ./test/unit",
-    "test:functional": "jest ./test/functional",
+    "test:functional": "jest ./test/functional --coverage=false",
```

Comment:
Couldn't get this to run locally, assume there is some additional setup required. I'd say it'd be worth updating the README with some instructions to run these tests locally (I could reverse engineer it from the workflow, but should be documented).
As some future followup, might be worth updating this so it works? Or changing the name to reflect that it should only run on CI, e.g `ci:functional` or similar.

Reply:
Useful feedback. The readme is now updated via most recent commit. The second part about ci:functional - makes perfect and will be addressed in the future as the pipeline might still change.